src/deploy/osp_deployer/settings/sample_xsp_profile.ini

```diff
@@ -525,6 +525,19 @@ powermax_array=CHANGEME
 powermax_port_groups=CHANGEME
 powermax_srp=CHANGEME
 
+
+#
+# PowerMax manila parameters.
+#
+enable_powermax_manila_backend=false
+manila_powermax_driver_handles_share_servers=true
+manila_powermax_nas_login=CHANGEME
+manila_powermax_nas_password=CHANGEME
+manila_powermax_nas_server=CHANGEME
+manila_powermax_server_container=CHANGEME
+manila_powermax_share_data_pools=CHANGEME
+manila_powermax_ethernet_ports=CHANGEME
+
```

Comment:
Probably need to update the comments section above to describe how to use this settings, like the rest of backends

Reply:
Yes, will do

Reply:
> Probably need to update the comments section above to describe how to use this settings, like the rest of backends

Done